hedera-node/docs/system-accounts-operations.md

```diff
@@ -161,13 +162,15 @@ _Please note that the expected behavior described in this section is valid if th
     - success, if address is empty + creates the account;
     - fail, if address is a non-payable contract.
 
-**Hedera:**
+**Hedera:** <a id="hedera-transfer-hts"/>
 1. For addresses in range 0x0 → 0x2EE (0.0.0 → 0.0.750):
-    - Transfers using `.send` and `.transfer`: fail with status `INVALID_FEE_SUBMITTED`. Exception: `tokenCreate` to address 0x167 which is successful.
+    - Transfers using `.send` and `.transfer`: fail with status `INVALID_FEE_SUBMITTED`. **Exception: `tokenCreate` to the HTS system contract 0.0.359 (`0x167`) is allowed[^1].**
     - Transfer of HTS tokens through the HTS system contract: fail with status `INVALID_RECEIVING_NODE_ACCOUNT`.
 2. For addresses in range 0x2EF → 0x3E8 (0.0.751 → 0.0.1000):
     - success, if account exists and has `receiverSigRequired` == false;
     - success, if account exists and has `receiverSigRequired` == false and account is `sender`;
+    
+[^1]:  Transfer to the HTS system contract on token create is so that additional fees, beyond what's reasonable to put in gas (e.g., due to the gas throttle limit), can be paid.
```

Comment:
Does this help?
```suggestion
[^1]:  Transfer to the HTS system contract on create token methods is allowed so that the additional HAPI fees  beyond the gas execution fee can be paid. This limitation exists due to the gas throttle limit which would be exceeded if the HAPI token entity create fee was converted to gas.
```